x-pack/qa/smoke-test-plugins-ssl/build.gradle

```diff
@@ -74,18 +70,13 @@ integTestCluster {
   extraConfigFile clientKey.name, clientKey
   extraConfigFile clientCert.name, clientCert
 
-  setupCommand 'setupTestUser',
-               'bin/elasticsearch-users', 'useradd', 'test_user', '-p', 'x-pack-test-password', '-r', 'superuser'
-  setupCommand 'setupMonitoringUser',
-               'bin/elasticsearch-users', 'useradd', 'monitoring_agent', '-p', 'x-pack-test-password', '-r', 'remote_monitoring_agent'
+  user username: "test_user", password: "x-pack-test-password"
+  user username: "monitoring_agent", password: "x-pack-test-password", role: "remote_monitoring_agent"
 
-  waitCondition = { NodeInfo node, AntBuilder ant ->
-    WaitForHttpResource http = new WaitForHttpResource("https", node.httpUri(), numNodes)
-    http.setTrustStoreFile(clientKeyStore)
-    http.setTrustStorePassword("testclient")
-    http.setUsername("test_user")
-    http.setPassword("x-pack-test-password")
-    return http.wait(5000)
+  project(':plugins').getChildProjects().each { pluginName, pluginProject ->
```

Comment:
Why is this necessary if we are using the default distribution?

Reply:
The default distribution has all the modules and the x-pack plugins, but  doesn't have the OSS plugins  ( there's a handful of those in `:plugins` ) for different specialized features, so the test essentially verifies that these can be installed.